### PR TITLE
chore: spring sdk integration tests

### PR DESCRIPTION
**Pull request by octonato**
Add integration tests (with testcontainers) for the Spring SDK. 

Once we merge this, we should remove the `samples/spring-wiring-tests`. Not removing now to avoid conflicts with #1084. 

Also, when we are ready to remove `samples/spring-wiring-tests`, we must expand the tests to include ValueEntity and Views. For now, I'm only add a simple test for Action. The goal is just to create the infra we need.

### CODE REVIEW

**Comment by octonato** [comment]
> Does it need something in the CI-config as well or does that already run IT test for all modules?

You are right. We need to do `sbt it:test` for the spring sdk. I will change

**Comment by octonato** [comment]
@johanandren, added it to CI. It will now run `IntegrationTest/test` for all projects, but only the SpringSDK have integration tests. 

If we add other IT tests in other submodules we will be covered already.

**Comment by octonato** [comment]
> @johanandren, added it to CI. It will now run `IntegrationTest/test` for all projects, but only the SpringSDK have integration tests.
> 
> If we add other IT tests in other submodules we will be covered already.

That didn't work out. To use testcontainers, we need to change the setup of the job. To avoid changing the main test job, I moved the spring IT tests to its own job.

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ commands:
           parent: spring-sdk # we need to revert this when merging back into main
 
   install-java-11:
-    description: install openjdk-11
+    description: install openjdk-11t
```
Comment (octonato) [inline .circleci/config.yml:20]: Oeps!

**Review by johanandren — APPROVED** [review]
LGTM once that IT runs and passes :)

**Pull request merged**